common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -269,13 +283,34 @@ public void run() {
         }
     }
 
+    private static void logParameters(@NonNull String tag, @NonNull String correlationId,
+                                      @NonNull Object parameters, @Nullable String publicApiId) {
+        final String TAG = tag + ":" + parameters.getClass().getSimpleName();
+
+        //TODO - conversion of PublicApiId in readable form.
```

Comment:
Leaving this TODO intentionally? Do we have a tracking item? Or something you plan to resolve in this PR?

Reply:
Oh yes, will pin an ADO item